src/main/java/core/basesyntax/service/impl/CsvFormatReportGenerator.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class CsvFormatReportGenerator implements ReportGenerator {
```

Comment:
Don't begin class or method implementation with an empty line. It is recommended to remove line 2 to maintain a consistent code structure without redundant empty lines.